src/workerd/api/hibernatable-web-socket.h

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+
+#include <kj/debug.h>
+
+#include <workerd/io/worker-interface.capnp.h>
+#include <workerd/api/global-scope.h>
+#include <workerd/io/worker-interface.h>
+
+namespace workerd::api {
+
+class HibernatableWebSocketMessageEvent final: public ExtendableEvent {
+public:
+  explicit HibernatableWebSocketMessageEvent(v8::Isolate* isolate, rpc::EventDispatcher::HibernatableWebSocketMessageParams::Reader params);
```

Comment:
Style nitpick: We wrap at 100 characters.

See: https://github.com/capnproto/capnproto/blob/master/style-guide.md

Also be sure to read this if you haven't already: https://github.com/capnproto/capnproto/blob/master/kjdoc/tour.md